client/src/main/java/com/paypal/selion/annotations/WebTest.java

```diff
@@ -49,42 +51,48 @@
     /**
      * Provide additional capabilities that you may wish to add as a name value pair. Values of true or false will be
      * treated as Boolean capabilities unless you surround the value with '
-     * 
+     *
      * <pre>
      * {@literal @}Test
      * {@literal @}WebTest(additionalCapabilities={"key1:value1","key2:value2"})
      * public void testMethod(){
-     *     //UI navigation steps 
+     *     //UI navigation steps
      * }
      * </pre>
      */
     String[] additionalCapabilities() default {};
 
     /**
+     * Additional {@link DefaultCapabilitiesBuilder}s that this web test should use. Capabilities which are
+     * constructed this way are merged with the results of any/all globally applicable capability builders.
+     */
+    Class<? extends DefaultCapabilitiesBuilder>[] additionalCapabilitiesBuilders() default {};
```

Comment:
in most cases, no it is not required --  So this works;

```java 
@WebTest( additionalCapabilitiesBuilders = MyClass.class )
```

However, if a user wants to have some sort of separation of concerns, the array here can help.  

For example, let's say a user that wants their own SauceCapabilitiesBuilder and ChromeCapabilitiesBuilder.. Furthermore, they only want their SauceCapabilitiesBuilder to do something when SeLion is configured to use sauce labs or the ChromeCapabiltiesBuilder to do something when SeLion is configured to use chrome. In this example, they'd could not utilize a single `@WebTest` annotated test method and have it work as expected, based on runtime configuration, unless we had array support.

```java
@WebTest( additionalCapabilitiesBuilders = { MySauceCapabilitiesBuilder.class, MyChromeCapabilitiesBuilder.class } )
```

- `-DSELION_USE_SAUCE_LABS=true` could cause the `MySauceCapabilitiesBuilder` to apply some capabilities
- `-DSELION_SELENIUM_BROWSER=chrome` could cause the `MyChromeCapabilitiesBuilder` to apply some capabilities